feature_tests/js/api/Foo.js

```diff
@@ -24,4 +24,9 @@ export class Foo {
   get_bar() {
     return new Bar(wasm.Foo_get_bar(this.underlying), true, [this]);
   }
+
+  static new_static(arg_x) {
+    const buf_arg_x = diplomatRuntime.DiplomatBuf.str(wasm, arg_x);
+    return new Foo(wasm.Foo_new_static(buf_arg_x.ptr, buf_arg_x.size), true, []);
+  }
```

Comment:
What's stopping `buf_arg_x` from being cleaned up here? Shouldn't it be added as an edge?

Reply:
Ok, I misunderstood how this works. We'd need to add an edge to something with infinite lifetime, which I'm not sure how to obtain.

Reply:
We could add a new method to `DiplomatBuf` in `runtime.mjs` that unregisters it without freeing it, and it would look just like the `.free()` method but slightly different. I'm personally against this and feel like there's a better solution, but that's how it would be done...

Reply:
Or we could never register it?

Reply:
We could avoid ever registering it too, yeah. Would that be any better besides being slightly more efficient? Because adding a `.leak()` method that just unregisters it would be so easy, and since we're explicitly leaking I _hope_ that we never call it enough to be measurably slow...

Reply:
lmk if you need help implementing either of these